spec/ndx-ophys-devices.extensions.yaml

```diff
@@ -1,8 +1,201 @@
 groups:
-- neurodata_type_def: TetrodeSeries
-  neurodata_type_inc: ElectricalSeries
-  doc: An extension of ElectricalSeries to include the tetrode ID for each time series.
+- neurodata_type_def: DeviceModel
+  neurodata_type_inc: Device
+  doc: Extends Device to hold model metadata.
   attributes:
-  - name: trode_id
-    dtype: int32
-    doc: The tetrode ID.
+  - name: model
+    dtype: text
+    doc: Model of the device.
+    required: false
+- neurodata_type_def: Indicator
+  neurodata_type_inc: Device
+  doc: Extends Device to hold metadata on the Indicator.
+  attributes:
+  - name: label
+    dtype: text
+    doc: Indicator standard notation.
+  - name: injection_location
+    dtype: text
+    doc: Injection brain region name.
+    required: false
+  - name: injection_coordinates_in_mm
+    dtype: float
+    shape:
+    - 3
+    doc: Indicator injection location in stereotactic coordinates (AP, ML, DV) mm
+      relative to Bregma.
+    required: false
+- neurodata_type_def: OpticalFiber
+  neurodata_type_inc: DeviceModel
+  doc: Extends DeviceModel to hold metadata on the Optical Fiber.
+  attributes:
+  - name: numerical_aperture
+    dtype: float
+    doc: Numerical aperture, e.g., 0.39 NA.
+    required: false
+  - name: core_diameter_in_um
+    dtype: float
+    doc: Core diameter in micrometers.
+    required: false
+- neurodata_type_def: ExcitationSource
+  neurodata_type_inc: DeviceModel
+  doc: Extends DeviceModel to hold metadata on the Excitation Source.
+  attributes:
+  - name: illumination_type
+    dtype: text
+    doc: 'Type of illumination. Suggested values: LED, Laser or more specifically
+      Gas, Metal vapor, Solid state, Dye, Semiconductor, Free electron, etc.'
```

Comment:
```suggestion
    doc: 'Type of illumination. Suggested values: LED, Laser, Metal vapor, Solid state, Dye, Semiconductor, Free electron, etc.'
```

'or more specifically Gas' reads out of place here; when people see 'suggested values' or 'examples' they tend to copy/paste straight out of it

So did you mean to say there should be `Laser` as well as `Gas laser`?

Reply:
Yes, you're right! I will be more specific for the examples here